package.json

```diff
@@ -38,17 +42,27 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@types/chai": "~4.2.22",
+    "@types/chai-as-promised": "~7.1.8",
+    "@types/cookie-parser": "~1.4.6",
     "@types/express": "^4.17.13",
     "@types/jest": "^27.4.1",
     "@types/node": "^18.11.18",
     "@types/nunjucks": "^3.2.1",
+    "@types/sinon": "~7.5.2",
+    "@types/sinon-chai": "~3.2.12",
     "@types/supertest": "^2.0.11",
     "@typescript-eslint/eslint-plugin": "^5.16.0",
     "@typescript-eslint/parser": "^5.16.0",
+    "chai": "~4.2.0",
+    "chai-as-promised": "~7.1.1",
```

Comment:
How are these being used? the tests are using Jest so do you really need to add chai/sinon ? if not I'd suggest removing them.

Reply:
thanks @JoelClemence I have removed chai dependencies.